changelog/4774.bugfix.rst

```diff
@@ -0,0 +1 @@
+Requests to ``/model/train`` do not longer block other requests to the Rasa server.
```

Comment:
Just had this thought: what happens if I call this endpoint and then call it immediately again, while the previous request has not finished?

Reply:
it will train a second model. Since this doesn't store any information in global variables, we should be fine.